rmgpy/rmg/input.py

```diff
@@ -313,7 +313,7 @@ def model(toleranceMoveToCore=None, toleranceMoveEdgeReactionToCore=numpy.inf,to
           toleranceMoveEdgeReactionToSurface=numpy.inf, toleranceMoveSurfaceSpeciesToCore=numpy.inf, toleranceMoveSurfaceReactionToCore=numpy.inf,
           toleranceMoveEdgeReactionToSurfaceInterrupt=None,
           toleranceMoveEdgeReactionToCoreInterrupt=None, maximumEdgeSpecies=1000000, minCoreSizeForPrune=50, 
-          minSpeciesExistIterationsForPrune=2, filterReactions=False, ignoreOverallFluxCriterion=False,
+          minSpeciesExistIterationsForPrune=2, filterReactions=False, filterThreshold=1e8, ignoreOverallFluxCriterion=False,
```

Comment:
This commit is way out of order as the systems using this input were defined many commits ago.

Reply:
Are you certain you didn't say in the documentation that the default was 5e8?

Reply:
For PRs, I think GitHub lists commits by author date rather than commit date, which is rather inconvenient. It's possible that the actual commit order makes more sense.

Reply:
For PRs, I think GitHub lists commits by author date rather than commit date, which is rather inconvenient. It's possible that the actual commit order makes more sense.